install_config/upgrading/manual_upgrades.adoc

```diff
@@ -744,6 +974,119 @@ IMAGE_VERSION=<tag>
 +
 Replace `<tag>` with `{latest-tag}` for the latest version.
 
+. Configure Dnsmasq. Because of changes made in {product-title} 3.6, you will need to
+perform the following steps to configure dnsmasq as part of the upgrade. See DNS Changes
+under xref:../../release_notes/ocp_3_6_release_notes.adoc#ocp-36-notable-technical-changes[Notable Technical Changes]
+in the {product-title} 3.6 Release Notes.
++
+.. Download the following script:
++
+----
+# wget https://raw.githubusercontent.com/openshift/openshift-ansible/release-3.6/roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh -O /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
+# chmod 755 /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
+----
++
+This command downloads the *99-origin-dns.sh* script and copies the script to the *_/etc/NetworkManager/dispatcher.d/_* directory.
+This script configures pods to use the node IP address as resolver. Using 127.0.0.1 inside a pod would fail.
+
++
+.. Create a *_/etc/origin/node/node-dnsmasq.conf_*  node configuration file.
++
+----
+server=/in-addr.arpa/127.0.0.1
+server=/cluster.local/127.0.0.1
+----
++
+Sample *_node-dnsmasq.conf_* file:
++
+[source, bash]
+----
+$ cat /etc/origin/node/node-dnsmasq.conf
+  server=/in-addr.arpa/127.0.0.1
+  server=/cluster.local/127.0.0.1
+----
+
+.. Edit the *_/etc/dnsmasq.d/origin-dns.conf_* file as follows:
++
+----
+no-resolv
+domain-needed
+no-negcache
+max-cache-ttl=1
+enable-dbus
+bind-interfaces
+listen-address=<node_ip_address> <1>
+----
+<1> This is the IP address of the node host.
++
+Sample *_origin-dns.conf_* file:
++
+[source, bash]
+----
+$ cat /etc/dnsmasq.d/origin-dns.conf
+  no-resolv
+  domain-needed
+  no-negcache
+  max-cache-ttl=1
+  enable-dbus
+  bind-interfaces
+  listen-address=<node_ip_address>
+----
+
+.. Edit the *_/etc/dnsmasq.d/origin-upstream-dns.conf_* file as follows:
++
+----
+server=<dns_server1_ip_address>
+server=<dns_server2_ip_address>
+----
++
+Sample *_origin-upstream-dns.conf_* file:
++
+[source, bash]
+----
+$ cat /etc/dnsmasq.d/origin-upstream-dns.conf
+  server=<upstream-dns-server-1>
+  server=<upstream-dns-server-2>
+----
+
+.. Edit the *_/etc/origin/node/node-config.yaml_* as follows:
++
+----
+dnsBindAddress: 127.0.0.1:53
+dnsRecursiveResolvConf: /etc/origin/node/resolv.conf
+dnsDomain: cluster.local
+dnsIP: <node_ip_address> <1>
+----
+<1> This is the IP address of the node host.
++
+Sample *_node-config.yaml_* and *_resolv.conf_* files:
++
+[source, bash]
+----
+$ cat /etc/origin/node/node-config.yaml
+      ...
+    dnsBindAddress: 127.0.0.1:53
+    dnsRecursiveResolvConf: /etc/origin/node/resolv.conf
+    dnsDomain: cluster.local
+    dnsIP: <node-ip>
+    dockerConfig:
+      ...
+
+$ cat /etc/origin/node/resolv.conf
+  nameserver <upstream-dns-server-1>
+  nameserver <upstream-dns-server-1>
```

Comment:
You have `<upstream-dns-server-1>` twice. Please replace the second entry with `<upstream-dns-server-2>` - this was a typo in my bugzilla comment.